packages/manager/apps/web/client/app/dns-zone/history/dashboard/dashboard.html

```diff
@@ -126,13 +126,13 @@
                     data-on-dismiss="$ctrl.closeVisualizeDnsPopup()"
                 >
                     <oui-spinner ng-if="$ctrl.loadingDnsZoneData"></oui-spinner>
-                    <div
-                        style="white-space: pre-wrap;"
+                    <p
+                        style="white-space: pre-wrap;overflow: auto;max-height: 50vh"
```

Comment:
use max-height: 50dvh or 50svh would be better for mobile accessibility

Reply:
Thks I did not know this one :)